### PR TITLE
Migrate to GitHub actions

### PR DESCRIPTION
**Pull request by Alex0jk**
Add basic setup for GitHub actions : `release.yml` and `draft.yml`

This is for #30

### CODE REVIEW

**Review by sestrella — COMMENTED** [review]
@Alex0jk nice job so far! I would just suggest updating the status badges on the README file and also adding a section talking about the release process, if I understand this workflow correctly a maintainer would need to create a GH release before publishing a new version to Hackage. Am I right?

File: .github/workflows/README.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+```mermaid
+graph LR;
+    event[GH Event]-->|on push|Build;
+    event-->|tag created|Draft;
+    Draft-->|create draft release|End;
+    event-->|release published|Release;
+    Release-->|upload artifacts to Hackage (release candidate)|End;
+    Build-->End;
+```
```
Comment (CristhianMotoche) [inline .github/workflows/README.md:22]: @Alex0jk It seems this diagram has some kind of error. It cannot be rendered. Can you double check that please?

File: .github/workflows/draft.yml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+      - uses: addnab/docker-run-action@v3
+        with:
+          image: utdemir/ghc-musl:v24-ghc922
+          options: -v ${{ github.workspace }}:/mnt
+          run: |
+            cd /mnt
+            sh build.static.sh
+
+      - name: Change owner before compression
+        run: sudo chown $USER:$USER atomic-write
```
Comment (CristhianMotoche) [inline .github/workflows/draft.yml:24]: @Alex0jk atomic-write doesn't provide an executable. It's just a [library](https://github.com/stackbuilders/atomic-write/blob/master/atomic-write.cabal#L52). We don't need any step related to uploading a binary to a GH release.

**Review by CristhianMotoche — CHANGES_REQUESTED** [review]
@Alex0jk Looks good. Although, we don't have an executable on atomic-write so we don't need many of the steps that you defined in the `draft.yml` file. Also, can you give a look at the `README.md`? It seems the diagram is not rendering correctly. Thanks!

**Review by CristhianMotoche — APPROVED** [review]
LGTM!

**Pull request merged**